### ISSUE TITLE
SST Console Not Tracking Ion Deployments

### ISSUE THREAD

**Issue by prabal-rje** [sst/sst#4005]
Hello SST team,

1. I've noticed that the SST Ion console is not tracking deployments to AWS at all.
2. To alleviate this, I hit the 'rescan' button under 'Manage Workspace' > 'Accounts' > 'Rescan' (I had my account connected for use with earlier version of SST). I've mostly been using `bunx sst` instead of `sst`... however `sst version` yields 
3. Despite doing this, all my previous deployments show up as 'project_name/stage_name'... however, I am never able to see their resources. All I see is 'Waiting for resources…' perpetually.
4. I have no way to track my deployments to AWS any other way since SST Ion doesn't rely on CloudFormation (as it used to). What if I want to remove my deployments (repo's deleted because it was a 'test' deployment)?

Cheers,
PG

<img width="2551" alt="image" src="https://github.com/user-attachments/assets/99c4fdd3-92c0-4958-9d8f-b9d70b5a0817">

**Comment by jayair** [comment]
Yeah it could be. So are the deployments showing up?

**Comment by brendankerrison** [comment]
I have this issue as well. 

I have connected my account to the console, it finds the project and associated stages, however if I click on a stage it just hangs on "Waiting for resources…"

<img width="1722" alt="Screenshot 2024-11-04 at 1 37 47 PM" src="https://github.com/user-attachments/assets/3e0f1bbd-9b8f-4e2c-a83c-a493b1eeb456">

**Comment by prabal-rje** [comment]
> Yeah it could be. So are the deployments showing up?

No, not always. Also seeing something similar as the above user every once in a while. I have to fiddle around with it (removing, redeploying, forcing SST to re-scan my AWS account etc.) to get it to work (that is... if IT ends up working at all). This is all fine in 'dev' but it's not OK when it's also impacting me when deploying to 'production' stage.

<img width="1528" alt="image" src="https://github.com/user-attachments/assets/a07de3fd-59f7-47a9-ad08-b4b8eee77224">

**Comment by jayair** [comment]
We pushed out some updates this past week. Can you try syncing your AWS accounts again?

**Comment by brendankerrison** [comment]
This has resolved for me.
